src/views/Main/index.jsx

```diff
@@ -78,8 +79,15 @@ class MainContainer extends Component {
     }
 
     async getReportees(userSession, ldapEmail) {
-      const secretsClient = userSession.getTaskClusterSecretsClient();
-      const partialOrg = await getAllReportees(secretsClient, ldapEmail);
+      let partialOrg;
+      // For LDAP users
+      console.log('login type', userSession.loginType);
+      if (userSession.loginType === 'auth.mozilla.auth0.com') {
```

Comment:
You can check if `process.env.ALTERNATIVE_AUTH` is either true or false to determine if we're using the alternative auth that allows Github authentication for non-LDAP users.

Reply:
Thanks @armenzg. I'll try that